packages/core/usePictureInPicture/index.ts

```diff
@@ -0,0 +1,43 @@
+import { computed, ref } from 'vue-demi'
+
+import type { MaybeElementRef } from '../unrefElement'
+
+import { useEventListener } from '../useEventListener'
+
+export function usePictureInPicture(
+  el: MaybeElementRef<HTMLVideoElement | undefined>,
+) {
+  const video = ref(el)
+
+  const isSupported = computed(() => {
+    return 'pictureInPictureEnabled' in document && !video.value?.disablePictureInPicture
+  })
+
+  const pictureInPictureWindow = ref<PictureInPictureWindow | null>(null)
+
+  const isInPictureInPictureMode = computed(() => {
+    return !!pictureInPictureWindow.value
+  })
+
+  useEventListener(video, 'leavepictureinpicture', () => pictureInPictureWindow.value = null)
```

Comment:
Seems we need add `enterpictureinpicture` event

Reply:
What would be the need? We're using the `computed` value to detect if we're in Picture in Picture mode ...

Reply:
We should capture it when users toggle it without `togglePictureInPicture` function